cmd/server/main.go

```diff
@@ -131,8 +132,22 @@ func main() {
 	mcpService := mcp.NewService()
 	searchService := search.NewService(logger.WithComponent("search"))
 
+	taskService, err := task.NewService(
+		config.AppConfig.TemporalEndpoint,
+		config.AppConfig.TemporalNamespace,
+		config.AppConfig.TemporalAPIKey,
+		db.Queries,
+		logger.WithComponent("task"),
+	)
+	if err != nil {
+		log.Error("failed to initialize task service", slog.String("error", err.Error()))
+		// Don't exit, just disable task scheduling feature
+		log.Warn("task scheduling feature disabled")
```

Comment:
I think we should panic here, much easier to debug, no reason to launch the service if it's not fully working